pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -126,6 +127,8 @@ type jsonVM struct {
 		ProvisioningModel         string
 	}
 	MachineType string
+	// CPU platform corresponding to machine type; see https://cloud.google.com/compute/docs/cpu-platforms
+	CpuPlatform string
```

Comment:
Nit: probably more idiomatic to call this `CPUPlaform`: https://github.com/golang/go/wiki/CodeReviewComments#initialisms (and also more consistent with `CPUArch` and `CPUFamily`).